stateoftheuniverse/widgets/moon_functions.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+"""
+A function to return moon phase & moon brightness.
+"""
+
+import numpy as np
+import astroplan
+import astropy
+from astropy.time import Time
+import matplotlib.pyplot as plt
+from astropy.coordinates import get_moon, get_sun
+import datetime
+import warnings
+warnings.filterwarnings('ignore')
+
+def get_moon_illumination(time: datetime) -> float:
```

Comment:
Given how you currently import the `datetime` module, I believe the data type here actually would need to be `datetime.datetime`.

Reply:
Correct!